contracts/token/Minter.sol

```diff
@@ -95,14 +110,35 @@ contract Minter is Manager, IMinter {
      * @dev Set the reward token amounts for the round. Only callable by the RoundsManager
      */
     function setCurrentRewardTokens() external onlyRoundsManager whenSystemNotPaused returns (bool) {
+        setInflation();
+
         currentMintableTokens = mintedTokensPerRound();
         currentMintedTokens = 0;
-        currentRedistributableTokens = redistributableTokensPerRound();
+        currentRedistributableTokens = redistributionPool;
         currentRedistributedTokens = 0;
 
         return true;
     }
 
+    /*
+     * @dev Set inflation based upon the current bonding rate
+     */
+    function setInflation() internal {
+        uint256 currentBondingRate = (bondingManager().getTotalBonded() * PERC_DIVISOR) / livepeerToken().totalSupply();
+
+        uint256 newInflation = inflation;
+
+        if (currentBondingRate < targetBondingRate) {
+            // Bonding rate is below the target - increase inflation
+            newInflation = newInflation.add(inflationChange);
+        } else if (currentBondingRate > targetBondingRate) {
+            // Bonding rate is above the target - decrease inflation
+            newInflation = newInflation.sub(inflationChange);
```

Comment:
do we have to manually check for underflow? We don't want `.sub()` to throw if inflation goes to 0...we just want it to remain at 0.